packages/twenty-server/src/modules/workflow/common/standard-objects/workflow.workspace-entity.ts

```diff
@@ -18,11 +18,34 @@ import { STANDARD_OBJECT_IDS } from 'src/engine/workspace-manager/workspace-sync
 import { FavoriteWorkspaceEntity } from 'src/modules/favorite/standard-objects/favorite.workspace-entity';
 import { WorkflowEventListenerWorkspaceEntity } from 'src/modules/workflow/common/standard-objects/workflow-event-listener.workspace-entity';
 import { WorkflowRunWorkspaceEntity } from 'src/modules/workflow/common/standard-objects/workflow-run.workspace-entity';
-import {
-  WorkflowVersionStatus,
-  WorkflowVersionStatusOptions,
-  WorkflowVersionWorkspaceEntity,
-} from 'src/modules/workflow/common/standard-objects/workflow-version.workspace-entity';
+import { WorkflowVersionWorkspaceEntity } from 'src/modules/workflow/common/standard-objects/workflow-version.workspace-entity';
+
+export enum WorkflowStatus {
+  DRAFT = 'DRAFT',
+  ACTIVE = 'ACTIVE',
+  DEACTIVATED = 'DEACTIVATED',
+}
+
+const WorkflowStatusOptions = [
+  {
+    value: WorkflowStatus.DRAFT,
+    label: 'Draft',
+    position: 0,
+    color: 'yellow',
+  },
+  {
+    value: WorkflowStatus.ACTIVE,
+    label: 'Active',
+    position: 1,
+    color: 'green',
+  },
+  {
+    value: WorkflowStatus.DEACTIVATED,
+    label: 'Deactivated',
+    position: 2,
+    color: 'grey',
+  },
+];
```

Comment:
I think we should add a json constant that stores the valid status combinations:
```
export const validStatuses = {
draftActive: [WorkflowStatus.DRAF, WorkflowStatus.ACTIVE], 
draftDeactivated: [WorkflowStatus.DRAF, WorkflowStatus.DEACTIVATED], 
draft: [WorkflowStatus.DRAF], 
active: [WorkflowStatus.ACTIVE], 
deactivated [WorkflowStatus.DEACTIVATED]
}
```
Then we will need a utils that returns the validStatuses from statuses `export const getValidStatuses(statuses): validStatuses` that we can use in `workflow-update-job.ts`

Reply:
@martmull great feedback! I updated the code